src/libraries/System.Security.Principal.Windows/src/System.Security.Principal.Windows.csproj

```diff
@@ -59,8 +59,8 @@
              Link="Common\Interop\Interop.GetTokenInformation.cs" />
     <Compile Include="$(CommonPath)Interop\Windows\Advapi32\Interop.DuplicateTokenEx.cs"
              Link="Common\Interop\Interop.DuplicateTokenEx.cs" />
-    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.DuplicateHandle.cs"
-             Link="Common\Interop\Interop.DuplicateHandle.cs" />
+    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.DuplicateHandle.SafeAccessTokenHandle.cs"
```

Comment:
Nit: I believe that the usual naming convention under common/interop is to use underscore for the specialization, e.g.: `Interop.DuplicateHandle_SafeAccessTokenHandle.cs`